test/artifacts/DMMRouter02.json

```diff
@@ -0,0 +1,1147 @@
+{
+  "_format": "hh-sol-artifact-1",
+  "contractName": "DMMRouter02",
```

Comment:
Why it names as DMMRouter02? do we have DMMRouter01?

Reply:
For naming consistency, uniswap have router01 and router02 and we clone router02 only